tasks/src/app/ordered-list/ordered-list.component.css

```diff
@@ -0,0 +1,3 @@
+.highlightingName {
```

Comment:
```suggestion
.highlighting-name {
```
for css use kebab-case